.clang-format

```diff
@@ -0,0 +1,19 @@
+---
+Language:        Cpp
+BasedOnStyle:  Google
+ColumnLimit:     120
+#didn't we want to change this?
+NamespaceIndentation: All
+SortIncludes:    false
+IndentWidth:     2
+AccessModifierOffset: -2
+PenaltyBreakComment: 30
+PenaltyExcessCharacter: 100
+AlignAfterOpenBracket: Align
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+BinPackParameters: false
+AlwaysBreakTemplateDeclarations: MultiLine
```

Comment:
I thought we were going with
```
AlwaysBreakTemplateDeclarations: yes
```
?

Reply:
Thanks for testing it @davidlange6, but in the end I'd (also) prefer to stay with `yes`.